openmdao/docs/feature_reference/building_blocks/drivers/genetic_algorithm.rst

```diff
@@ -3,13 +3,13 @@
 Simple GA Driver
 ****************
 
-The SimpleGADriver is a driver based on a simple genetic algorithm implementation sourced from the lecture
-notes for the class 2009 AAE550 taught by Prof. William A. Crossley. The implementation is pure Python.
+.. note:: 
+    The SimpleGA is a driver based on a simple genetic algorithm implementation sourced from the lecture
```

Comment:
did you really mean to remove `Driver` from the name?  `SimpleGADriver` is a thing, `SimpleGA` is not?